src/components/tooltip/Tooltip.scss

```diff
@@ -47,8 +47,9 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
     }
 
     &.is-callout {
-        background-color: $bdl-purple-rain;
-        border-color: $bdl-purple-rain;
+        color: $bdl-dark-blue-02;
+        background-color: $bdl-dark-blue;
+        border-color: $bdl-dark-blue;
```

Comment:
double checking that these variables already exist (in case these are new colors)

Reply:
they exist 😄

Reply:
👍 😂